### ISSUE TITLE
lists.update_members requires "status" for users, when Mailchimp API does not

### ISSUE THREAD

**Issue by tgrant59** [charlesthk/python-mailchimp#68]
The "status" value is required by the function, but is not required by the Mailchimp API.

In my use case, I need to be able to update members' data without overwriting their status. The Mailchimp api allows the handy "status_if_new" parameter, which will update all other given fields and merge tags for a member without overwriting status.

I understand the reasoning for requiring "status" (the API call does nothing for a new user without it), but it breaks my use case. Instead, you could require either "status" or "status_if_new", or just drop the requirement entirely.

I can create a PR for either version. Which do you think is better? I think requiring just one of them is best.

**Comment by tgrant59** [comment]
This is also true for lists.members.create_or_update. It should allow the use of status_if_new.

There may be more places this applies

**Comment by stephenross** [comment]
I see that now, it appears the status_if_new option is not listed at all in the documentation outside of the example that they provided on the batch add. I will whip something up shortly, and I noticed there are a couple more functions to add that were added with the 2016-10-27 changelog that need to be implemented as well. If you would like to make a pull request to fix the issue with the status_if_new vs status checks, that would be great, but if you are short on time at the moment, I will get to it later today or sometime tomorrow.
